keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/ArtifactListener.kt

```diff
@@ -59,6 +60,8 @@ class ArtifactListener(
             .also { wasAdded ->
               if (wasAdded) {
                 publisher.publishEvent(ArtifactVersionUpdated(artifact.name, artifact.artifactType))
+                // send an event when the artifact is stored, so we can fetch its metadata
+                publisher.publishEvent(ArtifactSaved(artifact.copy(version = version), status))
```

Comment:
Any reason not to just have the event listener function subscribe to both events, instead of adding a second event here?

Reply:
I don't understand this comment - lets talk about it offline